### PR TITLE
CA-173358: Merge VBD related data sources to rrdp_iostat

### PR DESCRIPTION
**Pull request by kaifengz**
All VBD related data sources are merged from xcp-rrdd to rrdp_iostat.

Signed-off-by: Kaifeng Zhu <kaifeng.zhu@citrix.com>

### CODE REVIEW

**Comment by johnelse** [comment]
@xen-git test this please

**Comment by johnelse** [comment]
@kaifengz please could you break this commit down into a few smaller commits? It's hard to follow exactly what's changed with everything in one commit.

This code doesn't seem to be reading any metrics from /sys/devices (see [here](https://github.com/xapi-project/xcp-rrdd/blob/master/rrdd/xcp_rrdd.ml#L439-443) for the rrdd code which did this) - is this no longer needed?

**Comment by kaifengz** [comment]
@johnelse, I split the commit into three ones.
Felipe told me that ` /sys/devices/` is no longer in use, so I removed them when I was doing the merging.

**Comment by johnelse** [comment]
Thanks for the rework. Looks sensible; it's going in!
